test/spec/utils/utils.ex

```diff
@@ -10,7 +10,21 @@ defmodule SpecTestUtils do
   end
 
   def parse_yaml(list) when is_list(list), do: Enum.map(list, &parse_yaml/1)
+  def parse_yaml({"extra_data", x}), do: {:extra_data, parse_as_string(x)}
+  def parse_yaml({"transactions", list}), do: {:transactions, Enum.map(list, &parse_as_string/1)}
   def parse_yaml({k, v}), do: {String.to_atom(k), parse_yaml(v)}
   def parse_yaml("0x" <> hash), do: Base.decode16!(hash, [{:case, :lower}])
+  def parse_yaml("0x"), do: ""
```

Comment:
Maybe `<<>>` is more readable? I had to open a console to see that it was the same as "" in elixir xD

Reply:
This overload isn't actually needed, because `"0x" <> hash` matches `"0x"` 😅